### PR TITLE
port definition

### LINKED ISSUE #137: Rsync port is not used
**Describe the bug**
A port can be specfied but the rsync mover container does not receive the value

**Steps to reproduce**
Specify a port at the replicationsource wait for sync to occur

**Expected behavior**
Sync should use the specified port

**Actual results**
Port is not used 

**Additional context**

### PR DESCRIPTION
**Pull request by cooktheryan**
Signed-off-by: Ryan Cook <rcook@redhat.com>

**Describe what this PR does**
Resolves #137 

**Is there anything that requires special attention?**
N/A

**Related issues:**
#137

### CODE REVIEW

**Comment by cooktheryan** [comment]
Will update test coverage after manual testing

**Comment by cooktheryan** [comment]
```
port└─ $ oc logs -f -n source  scribe-rsync-src-database-source-zm8rb
Scribe rsync container version: 265a77e
Syncing data to 10.1.3.245:31280 ...
```

**Comment by cooktheryan** [comment]
```
main└─ $ oc logs -f scribe-rsync-src-uploader-pvc-q6p8d
Scribe rsync container version: 265a77e
Syncing data to 10.2.155.239:31697 ...

Number of files: 2 (dir: 2)
Number of created files: 0
Number of deleted files: 0
Number of regular files transferred: 0
Total file size: 0 bytes
Total transferred file size: 0 bytes
Literal data: 0 bytes
Matched data: 0 bytes
File list size: 0
File list generation time: 0.001 seconds
File list transfer time: 0.000 seconds
Total bytes sent: 79
Total bytes received: 20

sent 79 bytes  received 20 bytes  198.00 bytes/sec
total size is 0  speedup is 0.00
Rsync completed in 0s
Synchronization completed successfully. Notifying destination...
Initiating shutdown. Exit code: 0
```

File: controllers/replicationsource_test.go [hunk 1]
```diff
@@ -463,6 +464,35 @@ var _ = Describe("ReplicationSource", func() {
...
 
+	Context("when a port is defined", func() {
+		BeforeEach(func() {
+			remotePort := int32(2222)
+			remoteAddr := "my.remote.host.com"
+			rs.Spec.Rsync = &scribev1alpha1.ReplicationSourceRsyncSpec{
+				ReplicationSourceVolumeOptions: scribev1alpha1.ReplicationSourceVolumeOptions{
+					CopyMethod: scribev1alpha1.CopyMethodClone,
+				},
+				Address: &remoteAddr,
```
Comment (cooktheryan) [inline controllers/replicationsource_test.go:475]: @JohnStrunk I'm having issues here with having both address and Port defined. 
```
1 chart(s) linted, 0 chart(s) failed
/home/rcook/go/bin/golangci-lint run ./...
controllers/replicationsource_test.go:476: File is not `goimports`-ed (goimports)
				Port: &remotePort,
make: *** [Makefile:42: test] Error 1
```

Comment (JohnStrunk) [reply controllers/replicationsource_test.go:475]: Looks like it's a formatting issue, not syntax:
```
$ goimports -d controllers/replicationsource_test.go
diff -u controllers/replicationsource_test.go.orig controllers/replicationsource_test.go
--- controllers/replicationsource_test.go.orig	2021-03-22 15:00:15.260044777 -0400
+++ controllers/replicationsource_test.go	2021-03-22 15:00:15.260044777 -0400
@@ -473,7 +473,7 @@
 					CopyMethod: scribev1alpha1.CopyMethodClone,
 				},
 				Address: &remoteAddr,
-				Port: &remotePort,
+				Port:    &remotePort,
 			}
 		})
 		It("an environment variable is created", func() {
```

goimports (kind of) supersedes gofmt, so it requires standard go formatting of whitespace.

**Review by JohnStrunk — APPROVED** [review]

**Pull request merged**